cpt-repo.rst

```diff
@@ -356,6 +356,39 @@ repository life time.  Existing content will remain untouched, new content will
 be processed with the new settings.
 
 
+External Files
+^^^^^^^^^^^^^^
+
+Files in a CernVM-FS repository can be marked as "external files". Externals
```

Comment:
you can think about if replacing " around the word by cursive writing style looks nicer